.env.template

```diff
@@ -14,7 +14,6 @@ GEOSERVER_PORT=8088
 GEOSERVER_ADMIN_NAME=admin
 GEOSERVER_ADMIN_PASSWORD=geoserver
 
-WWW_HOST=http://localhost
 WWW_PORT=8080
 CESIUM_ACCESS_TOKEN=
```

Comment:
`WWW_PORT` and `CESIUM_ACCESS_TOKEN` missing from the EnvVariable class.

Reply:
This was intentional, since they are not used in the python codebase and are checked by docker-compose. Do you think I should stull add them in?

Reply:
> This was intentional, since they are not used in the python codebase and are checked by docker-compose. Do you think I should stull add them in?

If they serve no purpose within the python codebase, not including them makes sense to me. What do you think?